src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -85,6 +86,8 @@ public function createAction(Request $request): Response
             return $this->renderDisableMultistorePage();
         }
 
+        //@todo: form builder
+        $categoriesForm = $this->createForm(CategoriesType::class);
```

Comment:
We don't need a new form, categories are already part of the Product form and the categories data are also handled by the data provider